modules/hal_nordic/nrfx/nrfx_config.h

```diff
@@ -1056,10 +1056,12 @@
     #include <nrfx_config_nrf54h20_ppr.h>
 #elif (defined(NRF54H20_XXAA) || defined(NRF54H20_ENGB_XXAA)) && defined(NRF_FLPR)
     #include <nrfx_config_nrf54h20_flpr.h>
-#elif defined(NRF54L15_XXAA) && defined(NRF_APPLICATION)
-    #include <nrfx_config_nrf54l15_application.h>
-#elif defined(NRF54L15_XXAA) && defined(NRF_FLPR)
-    #include <nrfx_config_nrf54l15_flpr.h>
+#elif (defined(NRF54L05_XXAA) || defined(NRF54L10_XXAA) || defined(NRF54L15_XXAA)) \
+	&& defined(NRF_APPLICATION)
+    #include <nrfx_config_nrf54l_05_10_15_application.h>
```

Comment:
It's not autogenerated, is it? @anangl?

Reply:
If they are zero-diff, I'd consolidate them in a single file (templates are in HAL repo)

Reply:
They should be copied from https://github.com/zephyrproject-rtos/hal_nordic/tree/master/nrfx/templates which are auto-generated.

Reply:
These are anyway just templates, so it's not mandatory to use them exactly as they are provided.
In may opinion, it makes perfect sense to use one file instead of three if their actual content is the same. Wouldn't it be possible to somehow tweak this auto-generation (add an extra step or something) to end up with such common file as a template in nrfx?

Reply:
>These are anyway just templates, so it's not mandatory to use them exactly as they are provided.

Correct, but who is then responsible for keeping files generated and ones manually created here in sync? We've seen cases where incorrect/outdated config is causing issues.

I agree tweaking auto-generation is an option, but the gain is minimal. So for now I'd say we should use files provided.

Also, usage of nrfx_config in Zephyr seems to be overly complicated, I will propose a PR in the future to rework this.